### DISCUSSION TITLE
Current IP & VPN status integration with dash apps such as Homarr

### DISCUSSION THREAD

**Discussion by Kylar514** [qdm12/gluetun#1473] [Ideas]
Forgive me if it has been made already, the idea came to mind as I am setting up Homarr, and I am not sure of how it might be described/titled.  I tried searching it, but I havent seen anything, so perhaps this is a new idea.  Also not even sure if it would be a gluetun thing, Homarr thing, or a combination of both.

The idea is to have a widget or integration with dash apps, such that you can have a small widget that tells you what IP your VPN is using, and subsequently the status of the VPN.  

The way I test my VPN currently is by examining my torrent tracker, which displays the current IP of the torrent, so I suppose that this could be a way of implementing it.

**Comment by kayakyakr** [comment]
Bumping this, Gluetun has an api that could also be used, fetching `/v1/openvpn/status` gets you the status of the service.

https://github.com/qdm12/gluetun-wiki/blob/main/setup/advanced/control-server.md
